packages/flutter/lib/src/material/text_field.dart

```diff
@@ -332,13 +339,105 @@ class _TextFieldState extends State<TextField> {
     _editableTextKey.currentState?.requestKeyboard();
   }
 
-  void _onSelectionChanged(BuildContext context, SelectionChangedCause cause) {
+  void _handleSelectionChanged(TextSelection selection, SelectionChangedCause cause) {
     if (cause == SelectionChangedCause.longPress)
       Feedback.forLongPress(context);
   }
 
+  InteractiveInkFeature _createInkFeature(TapDownDetails details) {
+    final MaterialInkController inkController = Material.of(context);
+    final RenderBox referenceBox = InputDecorator.containerOf(_editableTextKey.currentContext);
+    final Offset position = referenceBox.globalToLocal(details.globalPosition);
+    final Color color = Theme.of(context).splashColor;
+
+    InteractiveInkFeature splash;
+    void onRemoved() {
+      if (_splashes != null) {
+        assert(_splashes.contains(splash));
+        _splashes.remove(splash);
+        if (_currentSplash == splash)
+          _currentSplash = null;
+        updateKeepAlive();
+      } // else we're probably in deactivate()
+    }
+
+    splash = Theme.of(context).splashFactory.create(
+      controller: inkController,
+      referenceBox: referenceBox,
+      position: position,
+      color: color,
+      containedInkWell: true,
+      borderRadius: BorderRadius.zero,
+      // TODO(hansmuller): splash clip border radius show match the input decorator's border.
```

Comment:
show->should?

Reply:
not sure i understand this todo :-)

Reply:
It belonged above the borderRadius parameter (I moved it). InkSplash and InkRipple use the borderRadius to modify the referenceBox sized clip when containedInkWell is true.  Outline borders (InputOutlineBorder) do have rounded corners and this parameter should match.